unified_planning/model/parameter.py

```diff
@@ -24,17 +24,17 @@
 class Parameter:
     '''Represents an action parameter or a fluent parameter.
     A parameter has a name, and a type.'''
-    def __init__(self, name: str, typename: 'up.model.types.Type', env: Environment = None):
+    def __init__(self, name: str, typename: 'up.model.types.Type', env: Environment):
```

Comment:
@Framba-Luca Sorry, my bad in the previous code review: can tou rever this change and leave environemnt as `Optional`?